backend/db_admin/app.py

```
@@ -0,0 +1,139 @@
+@app.route('/')
+def index():
+    """Redirect root to admin interface"""
+    return redirect(url_for('admin.index'))
+
+if __name__ == '__main__':
+    print("🚀 Starting SQLite Database Admin Panel...")
+    print("📊 Access at: http://localhost:5001")
+    print("🔗 Main API: http://localhost:8000")
+    print("💡 Press Ctrl+C to stop")
+    
+    app.run(host='0.0.0.0', port=5001, debug=True) 
```

Ruff (0.13.1)

Possible binding to all interfaces

(S104)